pkg-manager/plugin-commands-installation/src/installDeps.ts

```diff
@@ -116,6 +116,7 @@ export type InstallDepsOptions = Pick<Config,
   workspace?: boolean
   includeOnlyPackageFiles?: boolean
   prepareExecutionEnv: PrepareExecutionEnv
+  fetchFullMetadata?: boolean | null
```

Comment:
why is null needed? This already support `undefined`.
```suggestion
  fetchFullMetadata?: boolean
```

Reply:
I think [`? true : null`](https://github.com/pnpm/pnpm/pull/8569/commits/268a242863c4ffb0e13798b0c3b695ff78424f89) would be clearer than `?? true`.

Reply:
why would you use `?? true`? That wouldn't work correctly. You'd use `? true : undefined`

Reply:
I'm sorry, I meant `&& true`. (I got the symbol mixed up)

Reply:
ok, well use `? true : undefined`. That is less confusing.

Reply:
I chose instead to explicitly specify the type as `true | undefined` (https://github.com/pnpm/pnpm/pull/8569/commits/7cd1b894c71b328dff9977f7ef3b388ad0407e61).

I think `? true : undefined` would risk future me going back and do refactor or even somebody even change it to actual boolean due to misunderstanding `underfined` as `false`.